holidays/financial/national_stock_exchange_of_india.py

```diff
@@ -129,6 +173,11 @@ def _populate_public_holidays(self):
         for dt in self._add_eid_al_adha_day(tr("Bakri Id")):
             self._move_holiday(dt)
 
+        # Eid-E-Milad
+        if self._year in {2008, 2009}:
+            for dt in self._add_mawlid_day(tr("Id-E-Milad-Un-Nabi")):
+                self._move_holiday(dt)
```

Comment:
```suggestion
        if 2006 <= self._year <= 2009:
            # Prophet's Birthday.
            for dt in self._add_mawlid_day(tr("Id-E-Milad-Un-Nabi")):
                self._move_holiday(dt)
```
And this holiday should be placed after Ashura (chronologically).

Reply:
> 6  11-Apr-06  Tuesday  Mahavir Jayanti/ Id-E-Milad

Reply:
> And this holiday should be placed after Ashura (chronologically).

@saheelsapovadia, seems you missed that.

Reply:
❓